mpa/modules/models/heads/custom_cls_head.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import normal_init, constant_init, build_activation_layer
+
+from mmcls.models.builder import HEADS
+from mmcls.models.heads import LinearClsHead
+from mmcls.models.heads.cls_head import ClsHead
+
+
+@HEADS.register_module()
+class CustomNonLinearClsHead(ClsHead):
```

Comment:
Shouldn't this inherit from NonLinearClsHead, not ClsHead?

Reply:
Original one is from non_linear_cls_head.py
![image](https://user-images.githubusercontent.com/37075645/193715391-3e72d572-e1b8-4f29-8691-11de7adb24fb.png)

It inherit ClsHead.

Reply:
If you leave a non_linear_cls_head this should inherit from it.